crates/binjs_io/src/entropy/read/mod.rs

```diff
@@ -373,6 +379,10 @@ impl TokenReader for Decoder {
         path: &Path,
     ) -> Result<(InterfaceName, Option<std::rc::Rc<Box<[FieldName]>>>), TokenReaderError> {
         let name = main_stream!(self, interface_name_by_path, "interface_name_by_path", path)?;
+        // FIXME: How do we find out that we're expected to switch dictionary?
+        // FIXME: We need a mapping `InterfaceName` => `impl Node`, right?
+        // FIXME: Except that's not possible, we don't have a `Node` yet.
+        // FIXME: We should generate that code ourselves.
```

Comment:
I don't get this comment.
it's defined in the IDL and enter/leave is operated by generated code, right?

Reply:
Sorry, obsolete comment.